src/sentry/db/models/fields/slug.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models import SlugField
+
+from sentry.slug.validators import no_numeric_validator, org_slug_validator
+
+
+class SentrySlugField(SlugField):
+    default_validators = list(SlugField.default_validators) + [no_numeric_validator]
```

Comment:
`SlugField.default_validators` is already a list, are you converting it again to be safe?

Reply:
It is already a list true, but without creating a new list to append to, doing `.append()` on the original list will only extend `SlugField.default_validators`'s list. Does that make sense? haha

Reply:
This instantiates a new list that will get assigned to this class's attribute field

Reply:
that makes sense, but I thought doing `+` on two lists would return a new list rather than mutating `SlugField.default_validators` 🤔

Reply:
I was wrong initially. The type is of sequence, and not a list fully.![Screenshot 2024-01-02 at 1 47 17 PM](https://github.com/getsentry/sentry/assets/5581484/fa7c94fc-0fd4-4445-adda-b72d504c0811)

Reply:
I would use `[*SlugField.default_validators, no_numeric_validator]` personally